org_status/status_providers/__init__.py

```diff
@@ -17,6 +17,22 @@ class Status(Enum):
 }
 
 
+def get_status_provider_status(self):
+    raise NotImplementedError()
```

Comment:
Pretty sure this should be a method in the `StatusProvider` class, and not outside of it.

Reply:
Yep, needs to be in the `StatusProvider` class since you are looping through each StatusProvider and calling `get_status_provider_status`